docs/conversion/sakai_20_mysql_conversion.sql

```diff
@@ -8,4 +8,8 @@ UPDATE CM_OFFICIAL_INSTRUCTORS_T SET INSTRUCTOR_ID = LOWER(INSTRUCTOR_ID);
 
 ALTER TABLE POLL_OPTION ADD OPTION_ORDER INTEGER;
 
--- END SAK-41391
\ No newline at end of file
+-- END SAK-41391
+
+-- START SAK-41812
+ALTER TABLE SAKAI_PERSON_T ADD PHONETIC_PRONUNCIATION LONGTEXT DEFAULT NULL;
```

Comment:
Why not just a simple varchar(255)?

Reply:
Looks like @josecebe followed the same pattern of the HBM file here:
https://github.com/sakaiproject/sakai/blob/master/common/impl/src/java/org/sakaiproject/component/common/edu/person/SakaiPersonImpl.hbm.xml#L136

Defined as materialized_clob and Hibernate generates a LONGTEXT field.

Our client defined this field as varchar in oracle and failed, they needed to set a CLOB type.

Do you think it's worth changing the type in the HBM and the scripts?